### PR TITLE
chore(ci): converted travis workflow to github action

### PR DESCRIPTION
**Pull request by marcuslyons**
Ref [Level/community#99](https://github.com/Level/community/issues/99)

### CODE REVIEW

File: .github/workflows/sauce.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Sauce Labs
+on: push
+This conversation was marked as resolved by marcuslyons
+ Show conversation
```
Comment (vweevers) [inline .github/workflows/sauce.yml:4]: Copy-paste mistake?

File: .github/workflows/sauce.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        env:
+          # Download Sauce Connect binary now instead of on first run
+          SAUCE_CONNECT_DOWNLOAD_ON_INSTALL: true
+      - name: Add host
+        run: echo "127.0.0.1 airtap.local" | sudo tee -a /etc/hosts
+      - name: Test
+        run: npm run test-browsers
+        env:
+          SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
```
Comment (vweevers) [inline .github/workflows/sauce.yml:27]: We also have coverage data here so let's upload it, by adding the following steps at the end:

```yaml
- name: Coverage
  run: npm run coverage
- name: Codecov
  uses: codecov/codecov-action@v1
  with:
    file: coverage/lcov.info
```

File: package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "test": "standard && hallmark && (nyc -s node test.js | faucet) && nyc report",
     "test-browsers": "airtap --coverage --verbose test.js",
     "test-browsers-local": "airtap --coverage -p local test.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
```
Comment (vweevers) [inline package.json:14]: Can you run `npm rm coveralls --save-dev`? We no longer need it. Thanks!

File: .github/workflows/sauce.yml (hunk 2 above)
Comment (marcuslyons) [reply .github/workflows/sauce.yml:25]: Added in 96172d1

File: package.json (hunk 3 above)
Comment (marcuslyons) [reply package.json:14]: Added in 96172d1

File: .github/workflows/sauce.yml (hunk 1 above)
Comment (marcuslyons) [reply .github/workflows/sauce.yml]: Yeah, sorry about that.

Removed in 96172d1

**Pull request merged**